launchdarkly/README.md

```diff
@@ -1,22 +1,56 @@
-# LaunchDarkly Integration
+# LaunchDarkly integration
 
 ## Overview
 
-The Datadog-LaunchDarkly integration allows you to see the effects of your LaunchDarkly feature deployments on your customer's services or systems. For instance, if a deployed feature causes a service to slow down, you would be able to see the cause within Datadog.
+LaunchDarkly provides the following integrations with Datadog: 
 
-![LaunchDarkly events in Datadog][2]
+### Events integration
 
+The [LaunchDarkly][1] app for Datadog brings flag event markers to your monitoring dashboards, so you can see the effects of your LaunchDarkly feature deployments on your customer's services or systems. For instance, if a deployed feature causes a service to slow down, you can see the cause within Datadog. 
+
+### Dashboard widget
```

Comment:
@alexvetras, I decided to call this this "Dashboard widget" instead of the "LaunchDarkly app." Let me know if you would like me to change it back.